modules/model_loader.py

```diff
@@ -7,7 +7,7 @@ def load_file_from_url(
         *,
         model_dir: str,
         progress: bool = True,
-        file_name: str | None = None,
+        file_name: str = None,
```

Comment:
This is incorrect and introduces a mypy assignment error (_Incompatible default for argument_).